sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -117,7 +117,7 @@ case class CreateDataSourceTableCommand(table: CatalogTable, ignoreIfExists: Boo
  */
 case class CreateDataSourceTableAsSelectCommand(
     table: CatalogTable,
-    mode: SaveMode,
+    ignoreIfExists: Boolean,
```

Comment:
This is a good change, I like it! Now it is identical to the interface `CreateHiveTableAsSelectCommand`. Maybe we can copy the params here.